### PR TITLE
#1073: Remove guava

### PR DESCRIPTION
**Pull request by andreoss**
Per #1073 

- Remove guava as a test dependency

### CODE REVIEW

File: src/test/java/org/takes/rq/form/RqFormFakeTest.java [hunk 1]
```diff
@@ -76,6 +73,7 @@ public void createsFormRequestWithParams() throws Exception {
             key, avalue,
             akey, aavalue
         );
+        new RqPrint(req).printBody(System.err);
```
Comment (baudoliver7) [inline src/test/java/org/takes/rq/form/RqFormFakeTest.java:76]: @andreoss Why is it important to add this statement ?

**Review by baudoliver7 — CHANGES_REQUESTED** [review]
@andreoss Please, see my comment.

File: src/test/java/org/takes/rq/form/RqFormFakeTest.java (hunk 1 above)
Comment (andreoss) [reply src/test/java/org/takes/rq/form/RqFormFakeTest.java]: @baudoliver7 Removed (used it for debugging)

**Review by baudoliver7 — APPROVED** [review]
@andreoss It's ok for me.
@paulodamaso Please, check/merge.

**Comment by andreoss** [comment]
@baudoliver7 Forgot to remove todo, please take a look again.

**Review by baudoliver7 — APPROVED** [review]
@andreoss That's right ! Ok for me.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (128 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by sereshqua** [comment]
@baudoliver7 please make sure you will find at least 3 issues during next CR, thanks

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by baudoliver7** [comment]
@0crat status

**Comment by baudoliver7** [comment]
@0crat status

**Comment by baudoliver7** [comment]
@0crat status

**Comment by baudoliver7** [comment]
@0crat status

**Pull request merged**